### PR TITLE
Sqs consumer

### PR DESCRIPTION
**Pull request by jamesdools**
**Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      

**Describe what the PR does**    

**State whether the PR is ready for review or whether it needs extra work**    

**Additional context**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,9 @@
     "express": "^4.16.4",
     "ffmpeg-static-electron": "^2.0.1",
     "fluent-ffmpeg": "^2.1.2",
+    "lodash": "^4.17.11",
```
Comment (pietrop) [inline package.json]: Are actually using lodash?

File: config/default.json [hunk 2]
```diff
@@ -7,5 +7,8 @@
   },
   "aws": {
     "region": "eu-west-1"
+  },
+  "queue": {
+    "url": "SQS-url-here"
```
Comment (pietrop) [inline config/default.json]: In PSTT node sdk we have used a pattern of passing this through environment variables. Can we discuss the two approaches and have some consistency across the services?

File: app.js [hunk 3]
```diff
@@ -1,44 +1,28 @@
-const path = require('path');
 const config = require('config');
-const express = require('express');
```
Comment (pietrop) [inline app.js]: do we need express for cosmos instance? to return status 200 to indicate it's running?
see follow up comment below

File: package.json [hunk 4]
```diff
@@ -24,7 +24,9 @@
     "express": "^4.16.4",
```
Comment (pietrop) [inline package.json]: do we still need express?

File: app.js (hunk 3 above)
Comment (jamesdools) [reply app.js]: Good catch. Readding that in now!

File: config/default.json (hunk 2 above)
Comment (jamesdools) [reply config/default.json]: yep - we can discuss this tomorrow 👍

File: test/lib/messageHandler.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  beforeEach(() => {
+    sandbox.stub(converter, 'convertToWav').resolves(0);
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('calls the convertToWav for convert_audio job types', async () => {
+    const testMessage = _.cloneDeep(sqsMessage);
```
Comment (pietrop) [inline test/lib/messageHandler.js]: btw looks like you are using `lodash` for the deep clone

Comment (jamesdools) [reply test/lib/messageHandler.js]: 🤦‍♂ that's what it was

Comment (pietrop) [reply test/lib/messageHandler.js]: Ok, if it’s just in tests then bring it as dev dependency?

**Comment by pietrop** [comment]
@jamesdools it be good to get an update on this PR, is it ready to be tried it out in a deployment context? has it been tested in AWS already, and whether there is anything still outstanding before being able to merge?

File: config/default.json (hunk 2 above)
Comment (pietrop) [reply config/default.json]: leaving as separate PR

File: test/lib/messageHandler.js (hunk 5 above)
Comment (pietrop) [reply test/lib/messageHandler.js]: > keeping lodash in main dependencies a as could be useful in conjunction with sqs consumer message filtering

**Pull request merged**